torch_brain/data/collate.py

```
@@ -351,3 +351,19 @@ def collate(batch):
+    "chain",
+    "collate",
+    "pad",
+    "pad8",
+    "pad2d",
+    "pad2d8",
+    "track_batch",
+    "track_mask",
+    "track_mask8",
+    "track_mask2d",
+    "track_mask2d8",
+]
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)